tests/cloud_test_logic/cloud_test_index.py

```diff
@@ -110,5 +113,27 @@ class CloudTestIndex(str, Enum):
         "imagePreprocessing": {
             "patchMethod": "simple",
         }
-    }
+    },
+    CloudTestIndex.structured_languagebind_model: {
+        "type": "structured",
+        "model": "LanguageBind/Video_V1.5_FT_Audio_FT_Image",
+        "inferenceType": "marqo.GPU",
+        "storageClass": "marqo.performance",
```

Comment:
Why are we using performance?

Reply:
Thought it would be good for test coverage. I just switched to balanced